### ISSUE TITLE
its been a week and i still cant get any example code to work.

### ISSUE THREAD

**Issue by dosuserx** [adafruit/Adafruit-Feather-RP2040-USB-Host-PCB#1]
im not even sure what im doing wrong. all i know is that while i can upload code to the rp2040 usb host feather fine, after i have done so i cannot get the serial monitor to work. i tried using putty, but for some reason i have like 21 unnamed usb com ports. arduino 2 app will not recognize a port nor will windows name the port. i mainly just want to make a code that powers a usb type b midi device and transports the midi code to the usb c port. all the tutorials are written for older versions of windows and arduino. basically nothing is written about this board online. barely anything. does anyone have a suggestion of where to start with this board?

**Comment by BlitzCityDIY** [comment]
hihi- sorry you've been having problems. we recommend going to the forums for support questions like this for a quick response and troubleshooting assistance: https://forums.adafruit.com/viewforum.php?f=57

i was able to recreate your problems after updating the Philhower core. To resolve it, I closed out of the Arduino IDE, navigated to the Arduino packages folder on Windows (C:\Users\\[username]\AppData\Local\Arduino15\packages), deleted the rp2040 folder, reopened the Arduino IDE and installed the Philhower core so that it was a fresh install. After that, I uploaded the code to the board and was able to use the USB host simple example from TinyUSB and see device info.

i have not tested with the Arduino 2 IDE yet, so I would also recommend using version 1 to see if that solves the problem. if you are still unable to upload code to the board after trying these two ideas please post to the forums.

**Comment by dosuserx** [comment]
> hihi- sorry you've been having problems. we recommend going to the forums for support questions like this for a quick response and troubleshooting assistance: https://forums.adafruit.com/viewforum.php?f=57

is that not what i did? sorry if i posted this in the wrong place. ive never tried to get tech support like this before and i find it confusing. thank you for your assistance. ill try these things.

**Comment by dosuserx** [comment]
yeah, it doesn't seem to work on either versions of arduino. or at least the older version of arduino has the same results as using arduino 2.

**Comment by caternuson** [comment]
Adding forum post for tie in:
https://forums.adafruit.com/viewtopic.php?t=209070

**Comment by dosuserx** [comment]
thank you. i have no idea what im doing. the arduino stuff i understand, but forum etiquette i guess not.
